applications/matter_weather_station/Kconfig.sysbuild

```diff
@@ -4,18 +4,21 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+#### Radio core
 config NRF_DEFAULT_IPC_RADIO
-	default y if BOARD_THINGY53_NRF5340_CPUAPP
+	default y
+
+config NETCORE_IPC_RADIO_BT_HCI_IPC
+	default y if WIFI_NRF700X
+
+config NETCORE_IPC_RADIO_IEEE802154
```

Comment:
I thought you will fix this btw of the above change. I think here we should go with `default y if !WIFI_NRF700X`

Reply:
Right, I didn't notice that, will fix it.